migrations/2021-03-18-133300_create-favorites/up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE users (
+  token_id TEXT,
+  notifications int,
+  PRIMARY KEY (token_id)
+);
+
+CREATE TABLE favorites (
+  resturant_id int,
```

Comment:
`restaurant_id INTEGER NOT NULL`

tror vi skal bruge integer og ikke int. Men er ikke 100%
Se her: https://stackoverflow.com/questions/20289410/difference-between-int-primary-key-and-integer-primary-key-sqlite